sdk/iOS/OoyalaSkinSDK/ViewControllers/OOSkinFullScreenViewController.m

```diff
@@ -38,6 +38,10 @@ - (instancetype)init {
 
 #pragma mark - Override view controller functions
 
+- (BOOL)prefersHomeIndicatorAutoHidden {
+  return YES;
```

Comment:
Do we need to do some kind of check to see if this is an iPhoneX? If not, will this effect older phones like the 8.  What about future phones?

Reply:
Only iPhone X have home indicator. For previous versions this function won't be called.

Reply:
it will only be called for iPhone X @pilievOoyala